component-library/src/app/pages/input-documentation/input-doc-code.component.ts

```diff
@@ -456,47 +387,89 @@ export class InputDocCodeComponent implements OnInit, TranslatedPageComponent {
       inputControl?.enable();
     }
   }
+  determineErrorState(value: string, formGroup: FormGroup, formID: string) {
+    let errorArray: string[] = [];
+    switch (value) {
+      case 'Single':
+        errorArray = ['required'];
+        this.setErrors(formGroup, formID, errorArray);
+        break;
+      case 'Multiple':
+        errorArray = ['required', 'email', 'email2'];
+        this.setErrors(formGroup, formID, errorArray);
+        break;
+      case 'None':
+        errorArray = [];
+        this.setErrors(formGroup, formID, errorArray);
+        break;
+    }
+  }
+
+  setErrors(formGroup: FormGroup, formID: string, errorKeys: string[]) {
+    // eslint-disable-next-line
```

Comment:
Why is the linter off here?

Reply:
![Screen Shot 2023-09-13 at 11 53 20 AM](https://github.com/IRCC-ca/ds-sdc-dev/assets/111757720/d4c25348-2320-4544-8cd0-0a3a92324a2b)